_examples/opengl3_example/imgui-go_impl_glfw_gl3.go

```diff
@@ -9,6 +9,32 @@ import (
 	"github.com/inkyblackness/imgui-go"
 )
 
+// enum ImGuiKey_
+const (
+	ImGuiKeyTab        int = 0
```

Comment:
This whole list of constants could actually be in imgui package, as these are the identifier necessary for mapping to native keys.
When moving, please remove the "ImGui" prefixes, as they are already in package "imgui" then. Reading "imgui.ImGuiKeyPageDown" would stutter. Better have it "imgui.KeyPageDown" :)
And add the comment what these constants are for.

Furthermore, explicit int typing is not necessary here case.